### DISCUSSION TITLE
Weekly Sprint 9/10

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#109] [General]
Reporte de la reunión semanal.

1. Realizamos una revisión de nuestro estado actual, lo que hicimos y lo que deberíamos haber hecho. En la reunión semanal anterior (cuyo reporte no se publicó) habíamos quedado en terminar todas las maquetaciones para el domingo pero no pasó así que se va a dar hasta la mitad de la semana para terminar todas las maquetaciones.
2. Al sprint se le agregó la parte de back-end así que durante la semana vamos a tener reuniones cortas para avanzar en el diseño del modelo de datos del sistema.
3. Hubieron varias confusiones con respecto a responsables de las historias de usuario y responsable de las tareas así que se pasó a realizar la aclaración.
4. También se explicó cómo es el flujo de trabajo esperado, desde seleccionar la tarea, asignársela a uno mismo, ponerla en in progress, crear la rama, trabajar localmente, subir los cambios a la rama y crear el pull request.
5. También hablamos sobre cuáles páginas deberían tener bootstrap y en cuáles no son necesarias ya que al parecer no todas deben tenerlo.
6. También iremos creando discusiones para hablar sobre la fuente, sobre el logo y los colores sin que eso detenga el desarrollo del sistema.

**Comment by NataliaAlvarezIspc** [comment]
Gracias Roberto!
